components/service/fretboard/src/main/java/mozilla/components/service/fretboard/Experiment.kt

```diff
@@ -37,7 +37,12 @@ data class Experiment(
     /**
      * Experiment associated metadata
      */
-    val payload: ExperimentPayload? = null
+    val payload: ExperimentPayload? = null,
+    /**
+     * Last time the experiment schema was modified
```

Comment:
nit: Looks like here is an empty line missing before the doc comment?

Reply:
I think now it should be fine...